### DISCUSSION TITLE
Running slow on laravel vapor

### DISCUSSION THREAD

**Discussion by CovertError** [livewire/livewire#5157] [1. Help]
is there away to speed livewire when running on laravel vapor on our testing environment running locally livewire is fast but when it comes to laravel vapor its just takes alot of time any ideas of how to fix that?

**Comment by joshhanley** [comment]
@CovertError that's really hard to say without any details. You will need to do an analysis to work out what exactly is slow. Is it the network latency time, lamba runtime speed, cold vs warm container, database speed, etc.? Could be a variety of reasons for it taking longer.

Once you have investigated all that and the numbers are reasonable, next step is to look at your app performance in terms of how you have written it. For example are you using `.defer` on all your inputs (that way a request isn't sent on every keystoke), etc. Looking at the number of network requests being sent and reducing that amount will also make your app feel more performant.

If you can do an investigation and tell us what exactly is being slow, maybe we can provide some suggestions 🙂
Hope this helps!
